sdk-tests/src/test/java/io/dapr/it/actors/ActorExceptionIT.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) Microsoft Corporation and Dapr Contributors.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.it.actors;
+
+import io.dapr.actors.ActorId;
+import io.dapr.actors.client.ActorProxy;
+import io.dapr.actors.client.ActorProxyBuilder;
+import io.dapr.it.BaseIT;
+import io.dapr.it.actors.app.MyActor;
+import io.dapr.it.actors.app.MyActorService;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.ExecutionException;
+
+import static io.dapr.it.Retry.callWithRetry;
+import static io.dapr.it.TestUtils.assertThrowsDaprException;
+import static io.dapr.it.TestUtils.assertThrowsDaprExceptionSubstring;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class ActorExceptionIT extends BaseIT {
+
+  private static Logger logger = LoggerFactory.getLogger(ActorExceptionIT.class);
+
+  @Test
+  public void exceptionTest() throws Exception {
+    // The call below will fail if service cannot start successfully.
+    startDaprApp(
+        ActorExceptionIT.class.getSimpleName(),
+        MyActorService.SUCCESS_MESSAGE,
+        MyActorService.class,
+        true,
+        60000);
+
+    logger.debug("Creating proxy builder");
+    ActorProxyBuilder<MyActor> proxyBuilder =
+        new ActorProxyBuilder("MyActorTest", MyActor.class, newActorClient());
+    logger.debug("Creating actorId");
+    ActorId actorId1 = new ActorId("1");
+    logger.debug("Building proxy");
+    MyActor proxy = proxyBuilder.build(actorId1);
+
+    callWithRetry(() -> {
+      assertThrowsDaprExceptionSubstring(
+          "INTERNAL",
+          "INTERNAL: error invoke actor method: error from actor service",
```

Comment:
When testing as an integration test, we got the error message.

Reply:
This behavior is observed prior to this change as well.

Reply:
Seems like a fine error to me. Just curious, would the actual error be displayed anywhere? For instance, in the dapr runtime?

Reply:
It should be logged in Dapr runtime. The whole error handling for dapr is something that is not proper designed in the APIs today.